app/templates/view_users.html

```diff
@@ -25,132 +25,112 @@
-          {% if item.active %}
-            {% if current_user.has_any_role('admin', 'admin-ccs-category') %}
-              <form action="{{ url_for('.deactivate_user', user_id=item.id) }}" method="post">
-                  <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-                  <input type="hidden" name="source" value="{{ url_for('.find_user_by_email_address', email_address=email_address) }}"/>
-                  {%
-                      with
-                      type = "destructive",
-                      label = "Deactivate"
-                  %}
-                  {% include "toolkit/button.html" %}
-                  {% endwith %}
-              </form>
-            {% else %}
-              Active
-            {% endif %}
+        {% if item.active %}
+          {% if current_user.has_any_role('admin', 'admin-ccs-category') %}
+            <form action="{{ url_for('.deactivate_user', user_id=item.id) }}" method="post">
+              <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+              <input type="hidden" name="source" value="{{ url_for('.find_user_by_email_address', email_address=email_address) }}"/>
+              {{ govukButton({
+                "classes": "govuk-button--warning",
+                "text": "Deactivate"
+              }) }}
+            </form>
           {% else %}
-            {% if current_user.has_any_role('admin', 'admin-ccs-category') and not item.personalDataRemoved %}
-              <form action="{{ url_for('.activate_user', user_id=item.id) }}" method="post">
-                  <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
-                  <input type="hidden" name="source" value="{{ url_for('.find_user_by_email_address', email_address=email_address) }}"/>
-                  {%
-                      with
-                      type = "secondary",
-                      label = "Activate"
-                  %}
-                  {% include "toolkit/button.html" %}
-                  {% endwith %}
-              </form>
-            {% else %}
-              Deactivated
-            {% endif %}
+            Active
           {% endif %}
+        {% else %}
+          {% if current_user.has_any_role('admin', 'admin-ccs-category') and not item.personalDataRemoved %}
+            <form action="{{ url_for('.activate_user', user_id=item.id) }}" method="post">
+              <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+              <input type="hidden" name="source" value="{{ url_for('.find_user_by_email_address', email_address=email_address) }}"/>
+              {{ govukButton({
+                "classes": "govuk-button--secondary",
+                "text": "Activate"
+              }) }}
+            </form>
+          {% else %}
+            Deactivated
+          {% endif %}
+        {% endif %}
       {% endcall %}
 
-      {% endcall %}
-      {% endcall %}
-  </div>
```

Comment:
I found this stray closing div tag after sorting out the indentation problems in this code.

Reply:
Nice spot, let's get rid of it 🔥